go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/ibm/scc-go-sdk
+module github.com/ibm-cloud-security/scc-go-sdk
```

Comment:
you shouldn't change the module name of any go module... it creates many problems

Reply:
previously this was under `ibm-cloud-security`, it was changed after we merged from main